update_ror.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import zipfile
+import os
+import re
+import boto3
+from s2aff.consts import ROR_VERSION
+# Make a GET request to the Zenodo API endpoint
+response = requests.get(
+    "https://zenodo.org/api/records/?communities=ror-data&sort=mostrecent")
+
+
+# Get the download URL of the most recent ROR record
+download_url = response.json()["hits"]["hits"][0]["files"][0]["links"]["self"]
+file_name = response.json()["hits"]["hits"][0]["files"][0]["key"]
+
+latest_version = re.sub('-ror-data.zip', '', file_name)
+
+current_version = ROR_VERSION
+print("Currently used version is " + current_version)
+print("Latest version is " + latest_version)
+if latest_version == current_version:
+    print("ROR up to date. No action required.")
+else:
+    print("Downloading latest version from s3")
```

Comment:
So if this run multiple times, it'll redownload every time because ROR_VERSION is not updated automatically, right?

Reply:
Oh I see, you moved the fixed one to `DEFAULT_ROR_VERSION`

Reply:
oh, good point. yes as is if it's run again from the same environment it'll redownload. I'll fix that.